lib/engines/with_migrations.rb

```diff
@@ -0,0 +1,24 @@
+module Engines
+  # It DRYies the common snippet to append engine's migrations to the main app
+  module WithMigrations
```

Comment:
Mh leave it in `lib`for now then, I'll investigate. But thanks for the abstraction! :clap: